keyvi/include/keyvi/dictionary/fsa/internal/lru_generation_cache.h

```diff
@@ -68,21 +66,21 @@ class LeastRecentlyUsedGenerationsCache final {
 
   ~LeastRecentlyUsedGenerationsCache() {
     delete current_generation_;
-    for (MinimizationHash<EntryT>* generation : generations_) {
+    for (MinimizationHash<EntryT> *generation : generations_) {
       delete generation;
     }
   }
 
   LeastRecentlyUsedGenerationsCache() = delete;
-  LeastRecentlyUsedGenerationsCache& operator=(LeastRecentlyUsedGenerationsCache const&) = delete;
-  LeastRecentlyUsedGenerationsCache(const LeastRecentlyUsedGenerationsCache& that) = delete;
+  LeastRecentlyUsedGenerationsCache &operator=(LeastRecentlyUsedGenerationsCache const &) = delete;
+  LeastRecentlyUsedGenerationsCache(const LeastRecentlyUsedGenerationsCache &that) = delete;
 
   /** Add this object.
    * @param key The key to add
    */
   void Add(EntryT key) {
     if (current_generation_->Size() >= size_per_generation_) {
-      MinimizationHash<EntryT>* newGeneration = NULL;
+      MinimizationHash<EntryT> *newGeneration = NULL;
```

Comment:
Not related to the PR but, as we are touching this line, maybe we can change `NULL` -> `nullptr` ?
So we have more modern C++-ish code gradually.